src/praisonai/praisonai/gateway/kanban_dispatcher.py

```
@@ -0,0 +1,389 @@
+                'PRAISONAI_KANBAN_BOARD': task.board,
+                'PRAISONAI_KANBAN_WORKER': self.worker_id,
+            })
+            
+            # Build command to execute task
+            # This could be configurable, but for now use a simple approach
+            cmd = self._build_execution_command(task)
+            
+            logger.info(f"Spawning worker for task {task.id}: {' '.join(cmd)}")
+            
+            # Start process
+            process = subprocess.Popen(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)